Zappy_IA_Report/NN/Code/Tutorial2.py

```
@@ -0,0 +1,25 @@
+import tensorflow as tf
+from tensorflow import keras
+import numpy as np
+import matplotlib.pyplot as plt
+
+data = keras.datasets.fashion_mnist
+(train_images, train_labels), (test_images, test_labels) = data.load_data()
+
+class_names = ['T-shirts/top', 'Trouser', 'Pullover', 'Dress', 'Coat', 'Sandal', 'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
```

Pylint (3.3.7)

[convention] Line too long (121/100)

(C0301)